src/Plugin/Mailchimp/Forms.php

```diff
@@ -4,28 +4,42 @@
 
 use AdCaptcha\Widget\AdCaptcha\AdCaptcha;
 use AdCaptcha\Widget\Verify\Verify;
+use AdCaptcha\Plugin\Plugin;
 
 use MC4WP_Form;
+use MC4WP_Form_Element;
 
-class Forms {
+class Forms extends Plugin {
 
     public function setup() {
         add_action( 'wp_enqueue_scripts', [ AdCaptcha::class, 'enqueue_scripts' ], 9 );
+        add_action( 'wp_enqueue_scripts', [ Verify::class, 'get_success_token' ] );
+        add_action( 'wp_enqueue_scripts', [ $this, 'block_submission' ], 9 );
+        add_filter( 'mc4wp_form_content', [ $this, 'add_hidden_input' ], 20, 3 );
         add_action( 'admin_enqueue_scripts', [ $this, 'form_preview_setup_triggers' ], 9 );
         add_filter( 'mc4wp_form_errors', [ $this, 'verify' ], 10, 2 );
         add_filter('mc4wp_form_messages', function($messages) {
             $messages = (array) $messages;
             $messages['invalid_captcha'] = [
                 'type' => 'error',
-                'text' => __( 'Incomplete captcha, Please try again.', 'adCAPTCHA' ),
+                'text' => __( 'Please complete the I am human box.', 'adcaptcha' ),
```

Comment:
Consider moving the `Please complete the I am human box.` to something more global. At some point we'll be i18n'ing it anyway